components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/api/impl/GroupImpl.java

```diff
@@ -58,7 +61,9 @@
     public Response createGroup(DeviceGroup group) {
         String owner = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUsername();
         if (group == null) {
```

Comment:
better doing the null check of group even before getting "owner" value initiated.